shell/platform/embedder/embedder.h

```diff
@@ -2582,9 +2582,23 @@ FlutterEngineResult FlutterEngineRunInitialized(
 //------------------------------------------------------------------------------
 /// @brief      Adds a view.
 ///
-///             This is an asynchronous operation. The view should not be used
-///             until the |add_view_callback| is invoked with an `added` of
-///             `true`.
+///             This operation notifies the engine to allocate resources and
+///             inform Dart about the view, and on success, schedules a new
+///             frame. Finally, it invokes |info.add_view_callback| with whether
+///             the operation is successful.
+///
+///             This operation is asynchronous; avoid using the view until
```

Comment:
I would use a different word than "avoid". The Dart app's `PlatformDispatcher.instance.views` won't have this view until the callback is invoked with an `added` of `true`. The Dart app is _unable_ to use the view until then.

Reply:
Although the Dart app is unable to use the view until then, the embedder can use it by sending events. (However your version has sentences with similar meaning so it's fine.)